packages/react-scripts/config/webpack.config.js

```diff
@@ -756,24 +756,26 @@ module.exports = function (webpackEnv) {
           // The formatter is invoked directly in WebpackDevServerUtils during development
           formatter: isEnvProduction ? typescriptFormatter : undefined,
         }),
-      new ESLintPlugin({
-        // Plugin options
-        extensions: ['js', 'mjs', 'jsx', 'ts', 'tsx'],
-        formatter: require.resolve('react-dev-utils/eslintFormatter'),
-        eslintPath: require.resolve('eslint'),
-        context: paths.appSrc,
-        // ESLint class options
-        cwd: paths.appPath,
-        resolvePluginsRelativeTo: __dirname,
-        baseConfig: {
-          extends: [require.resolve('eslint-config-react-app/base')],
-          rules: {
-            ...(!hasJsxRuntime && {
-              'react/react-in-jsx-scope': 'error',
-            }),
-          },
-        },
-      }),
+      // START: EVERLONG CHANGES
```

Comment:
I decided to leave the linting out for now, especially since we have the pre-commit linting on `league-web`. This linting lints **everything** and runs on both `yarn build` and `yarn start`, which is a pain because it takes a really long time. Plus, no linting errors are reported currently when we run `yarn build:prod`

for reference: `yarn build:prod`
- took 243s with `react-scripts` v4
- took 374s with our current `react-scripts`